internal/testutil/acc/search_index.go

```diff
@@ -18,7 +18,7 @@ func CheckDestroySearchIndex(state *terraform.State) error {
 		}
 		ids := conversion.DecodeStateID(rs.Primary.ID)
 		searchIndex, _, err := Conn().Search.GetIndex(context.Background(), ids["project_id"], ids["cluster_name"], ids["index_id"])
-		if err == nil && searchIndex != nil {
+		if err == nil && searchIndex != nil && searchIndex.Status != "IN_PROGRESS" { // index can be in progess for some seconds after delete is called
```

Comment:
This change was needed because a cluster definition is no longer present in the config, so the check destroy is called instantly after acceptance tests deletes the search index. When the cluster config was defined this would take several minutes and leave room for the search index to fully delete.